web/beacon-app/src/features/projects/components/ProjectTooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import { Trans } from '@lingui/macro';
+import * as Tooltip from '@radix-ui/react-tooltip';
+
+import HintIcon from '@/components/icons/hint';
+
+function ProjectTooltip() {
+  return (
+    <>
+      <Tooltip.Provider>
+        <Tooltip.Root>
+          <Tooltip.Trigger asChild>
+            <button>
+              <HintIcon />
+            </button>
+          </Tooltip.Trigger>
+          <Tooltip.Portal>
+            <Tooltip.Content
+              className="w-full max-w-[275px] rounded-md bg-secondary-slate p-4 text-sm text-white"
+              sideOffset={5}
+              align="start"
+            >
+              <p>
+                <Trans>
+                  A project is a collection of topics. Topics are event streams that your services,
+                  applications, or models can publish or subscribe to for real-time data flows. Each
+                  project requires an API key and secret. Secrets are known only to you. If you lose
+                  a secret, you lose the topic and data generated by the event stream permanently.
+                </Trans>
+              </p>
+            </Tooltip.Content>
+          </Tooltip.Portal>
+        </Tooltip.Root>
+      </Tooltip.Provider>
+    </>
+  );
```

Comment:
In the previous PR, i was talking about this kind of duplication. Maybe, creating in the `common` folder `HelpTooltip` component with content as props wouldn't be a better way to handle that?